lib/utils.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const LinkHeader = require('http-link-header');
+
+const getCrossOrigin = (url = '', host) => {
+    if (host && url.indexOf(`://${host}`) < 0) {
+        return 'crossorigin';
+    }
+    return '';
+};
+
+const getPreloadAttributes = ({
+    assetUrl,
+    host,
+    asAttribute,
+    corsCheck = false,
+    noPush = true // Disable HTTP/2 Push behaviour for now
```

Comment:
Will be enabled in future once cache digests spec is implemented in browsers.

Reply:
Then maybe replace the `for now` with `until digest spec is implemented by most browsers` to help future maintainers

Reply:
Will do.